doc/dox_comments/header_files/random.h

```diff
@@ -137,6 +137,36 @@ WOLFSSL_API int  wc_InitRng(WC_RNG*);
 */
 WOLFSSL_API int  wc_RNG_GenerateBlock(WC_RNG*, byte*, word32 sz);
 
+/*!
+    \ingroup Random
+
+    \brief Creates a new random number.
+
+
+    \return rng on success
+    \return NULL on error
+
+
+    \param rng random number generator initialized with wc_InitRng
```

Comment:
I don't see "rng" as a param to this function.  I see "nonce", "nonceSz", and "heap".  A new WC_RNG structure is returned though.